packages/react-instantsearch-hooks-dom/src/ui/Breadcrumb.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+
+import { cx } from './lib/cx';
+import { isModifierClick } from './lib/isModifierClick';
+
+import type { CreateURL } from 'instantsearch.js';
+import type { BreadcrumbConnectorParamsItem as BreadcrumbItem } from 'instantsearch.js/es/connectors/breadcrumb/connectBreadcrumb';
+
+export type BreadcrumbProps = React.HTMLAttributes<HTMLDivElement> & {
+  items: BreadcrumbItem[];
+  createURL: CreateURL<BreadcrumbItem['value']>;
+  onNavigate: (value: BreadcrumbItem['value']) => void;
+  rootComponent?: React.ReactNode;
+  separatorComponent?: React.ReactNode;
+};
+
+export function Breadcrumb({
+  items = [],
+  createURL,
+  onNavigate,
+  rootComponent: RootComponent = 'Home',
+  separatorComponent: SeparatorComponent = '>',
```

Comment:
Since those are JSX elements and not function components, we don't need to capitalize them.

Reply:
And since they're not components, the prop names might not be great.

Reply:
Changed to `rootElement` / `separatorElement` in f2542f4d.